common/src/main/java/org/mcaccess/minecraftaccess/MainClass.java

```diff
@@ -170,13 +170,11 @@ public static void setScreenReader(ScreenReaderInterface screenReader) {
     }
 
     public static void speakWithNarrator(String text, boolean interrupt) {
+        if (Strings.isNotEmpty(text) && Minecraft.getInstance().isWindowActive()) {
+            log.warn("The speaking of string %s with interrupt%s was suppressed".formatted(text, interrupt));
```

Comment:
Log calls already have builtin formatting.
```suggestion
            log.warn("The speaking of string \"{}\" with interrupt={} was suppressed", text, interrupt);
```